docs/page-controller.md

```diff
@@ -88,13 +87,16 @@ Registering a React-powered page is similar to connecting a PHP page, but with s
 
 Register pages with `wc_admin_register_page()` using these parameters:
 
-* `id` - Identifies the page with the controller. Required.
-* `parent` - Denotes the page as a child of `parent`. Used for breadcrumbs. Optional.
-* `title` - Page title. Used to build breadcrumbs. String or array of breadcrumb pieces. Required.
-* `path` - Page path (relative to `#wc-admin`). Used for identifying this page and for linking breadcrumb pieces when this page is a `parent`. Required.
-* `capability` - User capability needed to access this page. Optional (defaults to `manage_options`).
-* `icon` - Dashicons helper class or base64-encoded SVG. Optional.
-* `position` - Menu item position for parent pages. Optional. See: `add_menu_page()`.
+-   `id` - Identifies the page with the controller. Required.
+-   `parent` - Denotes the page as a child of `parent`. Used for breadcrumbs. Optional.
+-   `title` - Page title. Used to build breadcrumbs. String or array of breadcrumb pieces. Required.
+-   `path` - Page path (relative to `#wc-admin`). Used for identifying this page and for linking breadcrumb pieces when this page is a `parent`. Required.
+-   `capability` - User capability needed to access this page. Optional (defaults to `manage_options`).
+-   `icon` - Dashicons helper class or base64-encoded SVG. Optional.
+-   `position` - Menu item position for parent pages. Optional. See: `add_menu_page()`.
+-   'nav_args` - Arguments for registering items in WooCommerce Navigation.
+-   'nav_args[ 'order' ]` - Order number for presentation.
+-   'nav_args[ 'parent' ]`- Menu for item to fall under.`woocommerce`,`woocommerce-settings`,`woocommerce-analytics`, or another category added by an extension are available.
```

Comment:
Yes, it could actually! WC bookings places their "Settings" page in the new Nav's Settings hierarchy whereas its in the Bookings menu in the traditional sidebar.